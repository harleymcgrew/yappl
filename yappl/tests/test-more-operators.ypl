print_line (2 = 2);
print_line (2 = 3);
print_line (3 != 2);
print_line (3 != 3);
print_line !(3 = 2);
print_line (-2 < 2);
print_line (-2 < -2);
print_line (100 > 100);
print_line (100 > 50);
print_line (19 <= 19);
print_line (17 <= 19);
print_line (21 >= 21);
print_line (21 >= 20);
print_line ((1 != 2) or (5 = 5));
print_line ((3 != 0) and (8 != 1));
print_line ([2, 3] @ [4, 5]);
print_line (1 :: [2, 3])