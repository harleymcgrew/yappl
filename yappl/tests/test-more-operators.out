true
false
true
false
true
true
false
false
true
true
true
true
true
true
true
[2,3,4,5]
[1,2,3]
